packages/codebytes/src/helpers/index.ts

```diff
@@ -0,0 +1,43 @@
+import Uri from 'jsuri';
+
+import { trackUserClick } from '../libs/eventTracking';
+
+export type CodebyteOptions = {
+  clientName: string;
+  parentPage: string;
+  renderMode: string;
+};
+
+export enum CodebytesParams {
+  Language = 'lang',
+  Text = 'text',
+  CopyMode = 'copy-mode',
+  ClientName = 'client-name',
+  Page = 'page',
+  Mode = 'mode',
+}
+
+export const getOptions = (): CodebyteOptions => {
+  const currentUri = new Uri(window.location.href);
+
+  return {
+    clientName:
+      currentUri.getQueryParamValue(CodebytesParams.ClientName) || 'Unknown',
+    parentPage:
+      currentUri.getQueryParamValue(CodebytesParams.Page) || document.referrer,
+    renderMode: currentUri.getQueryParamValue(CodebytesParams.Mode) || '',
+  };
```

Comment:
if you use the native [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) api here, you don't need `jsuri` as a dependency, we've started removing `jsuri` in other places as well. 

```suggestion
  const currentUri = new URL(window.location.href);

  return {
    clientName:
      currentUri.searchParams.get(CodebytesParams.ClientName) || 'Unknown',
    parentPage:
      currentUri.searchParams.get(CodebytesParams.Page) || document.referrer,
    renderMode: currentUri.searchParams.get(CodebytesParams.Mode) || '',
  };
```

Reply:
Oh awesome